.github/workflows/ci.yaml

```diff
@@ -354,5 +349,16 @@ jobs:
           build-args: |
             SAML_IDP_METADATA_PATH=./config/idp-metadata-staging.xml
           push: true
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+          tags: |
+            ${{ env.IMAGE }}:sha-${{ env.GITHUB_SHA_SHORT }}, \
+            ${{ env.IMAGE }}:main, \
+            ${{ env.IMAGE }}:latest
+
+      - name: Deploy to Staging via Webhook
+        uses: navied/secure-actions-webhook@0.2.0
+        env:
+          HMAC_SECRET: ${{ secrets.WEBHOOK_STAGING_SECRET }}
+        with:
+          url: https://mycustomdomain-dev.senecacollege.ca/hooks/deploy
+          hmacSecret: ${{ env.HMAC_SECRET }}
+          data: '{"tag": "${{ env.IMAGE }}:sha-${{ env.GITHUB_SHA_SHORT }}"}'
```

Comment:
Don't send image here, only the new tag.

Reply:
Apart from this, and one small question looks good